LyricFy/ViewModels/HomeViewModel.swift

```diff
@@ -19,17 +19,28 @@ class HomeViewModel {
     
     func setupViewData() {
         guard let fetched = dataService.getProjects() else { return }
-        
+        self.projects.removeAll()
```

Comment:
Essa função é pra ser usada só uma vez quando a view Model for iniciada, e projects vai ser iniciada com [] então não precisa do remove.All()